services/requester/requester.go

```diff
@@ -236,7 +236,7 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 	var to string
 	if tx.To() != nil {
 		to = tx.To().String()
-		e.collector.EvmAccountCalled(to)
+		e.collector.EVMAccountInteraction(to)
```

Comment:
Not really related to this PR, but having this line here, it means that we miss interactions coming from COAs. We also miss out on interactions that might come from other EVM Gateway Operators.

Reply:
good point yeah, tbh this metric should be reworked quite a bit, what we will likely do is analyse the events and deduce more information about what's going on. I will open an issue for it, thanks for flagging.